File: apps/s-2-admin/src/api/bookingReference/BookingReferenceCreateInput.ts
import { BookingWhereUniqueInput } from "../booking/BookingWhereUniqueInput";

export type BookingReferenceCreateInput = {
  typeField: string;
  uid: string;
  meetingId?: string | null;
  meetingPassword?: string | null;
  meetingUrl?: string | null;
  booking?: BookingWhereUniqueInput | null;
  externalCalendarId?: string | null;
  deleted?: boolean | null;
};

File: apps/cactus-demo/src/APIs/BookingReference/Dtos/BookingReferenceCreateInput.cs
namespace CactusDemo.APIs.Dtos;

public class BookingReferenceCreateInput
{
    public int? Id { get; set; }

    public string TypeField { get; set; }

    public string Uid { get; set; }

    public string? MeetingId { get; set; }

    public string? MeetingPassword { get; set; }

    public string? MeetingUrl { get; set; }

    public string? ExternalCalendarId { get; set; }

    public bool? Deleted { get; set; }

    public Booking? Booking { get; set; }
}
